cfme/tests/services/test_catalog.py

```diff
@@ -5,31 +5,55 @@
 import cfme.tests.configure.test_access_control as tac
 from cfme import test_requirements
 from cfme.services.catalogs.catalog import CatalogsView
+from cfme.services.catalogs.catalog import DetailsCatalogView
+from cfme.utils.appliance.implementations.ui import navigate_to
 from cfme.utils.update import update
 
 pytestmark = [test_requirements.service, pytest.mark.tier(2)]
 
 
 @pytest.mark.rhel_testing
 @pytest.mark.sauce
-def test_catalog_crud(appliance):
+def test_catalog_crud(request, appliance):
     """
     Polarion:
         assignee: nansari
         casecomponent: Services
         initialEstimate: 1/8h
         tags: service
     """
+    # Create Catalog
     catalog_name = fauxfactory.gen_alphanumeric()
     cat = appliance.collections.catalogs.create(name=catalog_name, description='my catalog')
+    request.addfinalizer(cat.delete_if_exists)
 
     view = cat.create_view(CatalogsView)
     assert view.is_displayed
-    view.flash.assert_success_message('Catalog "{}" was saved'.format(catalog_name))
+    if appliance.version >= '5.11':
```

Comment:
I think this may be a bug, @niyazRedhat is checking with the dev on this. Name was displayed in 5.10 but in 5.11 it was removed. But on the delete message, it uses the description.